css/manual.css

```diff
@@ -18,6 +18,7 @@ body {
     font-size: 17px;
     color: #AEC5AD;
     background: rgba(20,88,121,1);
+    line-height: 1.4em;
```

Comment:
tweaking line-height based on `font-family` and `font-size` must almost always be done.